saleor/static/js_src/dashboard.jsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import DataGrid from "react-datagrid";
+import sorty from "sorty";
+
+class DataTable extends React.Component {
+    state = {
+        data: sort(this.props.data, this.props.sortInfo),
+        sortInfo: this.props.sortInfo
+    };
+
+    handleSortChange(sortInfo) {
+        this.setState({
+            sortInfo: sortInfo,
+            data: sort(this.state.data, sortInfo)
+        });
+    };
+
+    render() {
+        return <DataGrid
+            idProperty={this.props.columns[0]+""}
+            dataSource={this.state.data}
+            columns={this.props.columns}
+            sortInfo={this.state.sortInfo}
+            onSortChange={this.handleSortChange.bind(this)}
```

Comment:
Babel supports binding through `::` operator:

```
onSortChange={::this.handleSortChange}
```

(This is the same as `this::this.handleSortChange`.)